src/main/drivers/pitotmeter/pitotmeter_ms4525.c

```diff
@@ -113,7 +114,8 @@ bool ms4525Detect(pitotDev_t * pitot)
     uint8_t rxbuf[4];
     bool ack = false;
 
-    pitot->busDev = busDeviceInit(BUSTYPE_I2C, DEVHW_MS4525, 0, OWNER_AIRSPEED);
+    pitot->busDev = busDeviceInit(BUSTYPE_I2C, DEVHW_MS4525_0 + pitotmeterConfig()->ms4525_i2c_address, 0, OWNER_AIRSPEED);
```

Comment:
I just added the value configured in the parameter during sensor initialization.

![Captura de tela 2021-07-13 132159](https://user-images.githubusercontent.com/25625982/125489091-58cac6ec-8fdf-4578-83f6-ceb33f9de4f6.jpg)